priv/gettext/es/LC_MESSAGES/default.po

```diff
@@ -0,0 +1,365 @@
+## "msgid"s in this file come from POT (.pot) files.
+###
+### Do not add, change, or remove "msgid"s manually here as
+### they're tied to the ones in the corresponding POT file
+### (with the same domain).
+###
+### Use "mix gettext.extract --merge" or "mix gettext.merge"
+### to merge POT files into PO files.
+msgid ""
+msgstr ""
+"Language: es\n"
+"Plural-Forms: nplurals=2\n"
```

Comment:
I _think_ this is what we need here given the error in the tests (https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html):

```suggestion
"Plural-Forms: nplurals=2; plural=n != 1;"
```